tests/test_shelldriver.py

```diff
@@ -45,3 +47,18 @@ def test_run_with_timeout(self, target_with_fakeconsole, mocker):
         assert res == ['success']
         res = d.run("test")
         assert res == (['success'], [], 0)
+
+    def test_get_ip_addresses(self, target_with_fakeconsole, mocker):
+        fake_ip_addr_show = """
+18: br-lan.42    inet 192.168.42.1/24 brd 192.168.42.255 scope global br-lan.42\       valid_lft forever preferred_lft forever
+18: br-lan.42    inet6 fe80::9683:c4ff:fea6:fb6b/64 scope link \       valid_lft forever preferred_lft forever
+"""
```

Comment:
```suggestion
        fake_ip_addr_show = r"""
18: br-lan.42    inet 192.168.42.1/24 brd 192.168.42.255 scope global br-lan.42\       valid_lft forever preferred_lft forever
18: br-lan.42    inet6 fe80::9683:c4ff:fea6:fb6b/64 scope link \       valid_lft forever preferred_lft forever
"""
```
Otherwise invalid escape sequence warnings occur.

Reply:
Done, thanks.